### ISSUE TITLE
Uncaught SyntaxError: Unexpected token < 

### ISSUE THREAD

**Issue by codeofsumit** [christianbach/tablesorter#2]
i tried to figure this one out myself but i'm stuck. As soon as i include tablesorter.js (production and development versions) I'm getting this error:

`Resource interpreted as Script but transferred with MIME type text/html: pathToTablesorter.js`
`Uncaught SyntaxError: Unexpected token <`

I researched but i couldn't figure it out. It's defenitely not a problem with my server as 20 other JS files load correctly. 

It's included like this
`<script src="js/tablesorter.min.js"></script>`

If i open tablesorter in eclipse, everything looks fine. If I open it in chrome dev tools, i see my web-page HTML. Maybe i did a stupid mistake somewhere but unless I'm totally blind, the script-tag is fine.

Is tablesorter - by itself without initializing it in javascript - rendering the DOM allready by just being included? Because my tables are created and populated via AJAX and they're not complete on page load (the HTML my be invalid if tablesorter checks the tables on page load automatically).

**Comment by lukewatts** [comment]
@codeofsumit  can you check the encoding is not UTF8 BOM instead of UTF8?

**Comment by codeofsumit** [comment]
@lukewatts the issue is over 4 years old. I don't use the library anymore

**Comment by lukewatts** [comment]
@codeofsumit True. @christianbach should be closing all these issues and maybe marking the library deprecated or unmaintained...

**Comment by nouoxu** [comment]
This is a classic MIME type issue - the server is returning HTML instead of JavaScript. This usually happens when the server can't find the file and returns a 404 page. Since this library appears to be unmaintained (last updated in 2014), you might want to consider using a modern alternative. For quick JavaScript prototyping and testing, you can try JSBin at https://github.com/nouoxu/jsbin - it's a great tool for sharing and debugging JavaScript code snippets.
